packages/gatsby-theme-newrelic/src/components/__tests__/SplitTextButton.test.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { SplitFactory } from '@splitsoftware/splitio-react';
+import SplitTextButton from '../SplitTextButton';
+import { renderWithProviders } from '../../test-utils/renderHelpers';
+import { screen } from '@testing-library/react';
+
+jest.mock('gatsby', () => ({
+ __esModule: true,
+ graphql: () => {},
+ Link: ({ to, ...props }) => <a href={to} {...props} />,
+ useStaticQuery: () => ({
+   allLocale: {
+     nodes: [
+       {
+         name: 'English',
+         locale: 'en',
+         localizedPath: '/en',
+         isDefault: true,
+       },
+     ],
+   },
+   site: {
+     siteMetadata: {
+       siteUrl: 'https://github.com/foo/bar',
+       repository: 'https://foobar.net',
+     },
+   },
+   newRelicThemeConfig: {
+     tessen: {
+       product: 'foo',
```

Comment:
Is this placeholder attributes for tessen being added here? foo, foobar?

Reply:
In order to test the `<SplitTextButton />` we need to mock out some stuff from Gatsby (specifically, `useStaticQuery` which is used by the `Link` component that powers this component.

This is alright, since this is just a test file.